packages/near-utils/lib/index.ts

```diff
@@ -1,9 +1,18 @@
-import { near, Transaction } from '@liquality/types'
+import { near, SwapParams, Transaction } from '@liquality/types'
+import { validateValue, validateSecretHash, validateExpiration } from '@liquality/utils'
+
 import BN from 'bn.js'
 
+export { validateSecret, validateSecretAndHash } from '@liquality/utils'
 export { transactions, Account, InMemorySigner, providers, KeyPair, keyStores } from 'near-api-js'
 export { BN }
 
+export function validateSwapParams(swapParams: SwapParams) {
+  validateValue(swapParams.value)
+  validateSecretHash(swapParams.secretHash)
+  validateExpiration(swapParams.expiration)
```

Comment:
can there also be some reasonable validation for the address? Such that it is a string and within limits? Here's some paramters: https://nomicon.io/DataStructures/Account.html